core/controlplane/config/templates/cloud-config-worker

```diff
@@ -146,9 +146,13 @@ coreos:
         --cluster_dns={{.DNSServiceIP}} \
         --cluster_domain=cluster.local \
         --cloud-provider=aws \
-        --kubeconfig=/etc/kubernetes/worker-kubeconfig.yaml \
+        {{- if .Experimental.ClusterTLSBootstrap.Enabled }}
+        --experimental-bootstrap-kubeconfig=/etc/kubernetes/worker-bootstrap-kubeconfig.yaml \
+        {{- else }}
         --tls-cert-file=/etc/kubernetes/ssl/worker.pem \
-        --tls-private-key-file=/etc/kubernetes/ssl/worker-key.pem
+        --tls-private-key-file=/etc/kubernetes/ssl/worker-key.pem \
```

Comment:
Or is there any flag to alter where the kubelet writes issued keys and certs to `/etc/kubernetes/ssl` rather than `/var/run/kubernetes`? so that we don't need `if-else`s to vary mounts accordingly to whether the TLS bootstrap is enabled or not.